### ISSUE TITLE
Recent apps problem

### ISSUE THREAD

**Issue by romrumario** [KieronQuinn/TapTap#269]
I installed Tap Tap on a Xiaomi Redmi Note 8 Pro, and was working very well. After sometime use, I needed to go to recents, and nothis was there. I restarted device, and I think the first time I did that the problem was gone. However two days later, the same again. and even after restart recent apps not working. I uninstall Tap Tap, and the probelm is gone. So sad, because I love the app.
Something to help or a workaroud, I would apreciate!!! Tahnks.

**Comment by KieronQuinn** [comment]
This is likely fixed in 0.9 Beta, assuming it was caused by the gesture accessibility issue

**Comment by romrumario** [comment]
I'll try the new version!! Thanks for your kindness and congratulations for your really good work!!

**Comment by ryohandoko9056** [comment]
> This is likely fixed in 0.9 Beta, assuming it was caused by the gesture accessibility issue

Yes, it did repair. And on version 0.8 Beta, it was indeed working. But only on the first try.
It is running too on Beta 0.9 when I tested it, however only in the first attempt do I get feedback. I tested it around 6 times, but after that, There weren't any feedback at all. So here's the photo when I re-run the setup. It did work, and it did indicate that the application was running, but after that setup, The application didn't produce any feedback anymore. The only way to get it running is only by re-run the setup, after that, It is nothing. Even when the application us also running.
![Screenshot_2020-10-12-20-49-45-770_com.kieronquinn.app.taptap.jpg](https://user-images.githubusercontent.com/71830623/95753987-96ee6680-0ccc-11eb-810f-6bfd9bada0c6.jpg)
